userguide/tutorials/invoice_examples.adoc

```diff
@@ -0,0 +1,132 @@
+= Invoice Examples
+
+This document provides examples of different invoice scenarios. 
+
+== Prerequisites
+
+Ensure that you have gone through the https://docs.killbill.io/latest/userguide_subscription.html#components-invoicing[_Subscription User Guide_] and are familiar with the basic invoicing concepts.
+
+
+== Invoice Terminology - work in progress
```

Comment:
I want to give you some credit for trying to explain this, but I think it's harder than it looks and perhaps belongs more in our Subscription Manual / Glossary.